### DISCUSSION TITLE
GitHub Codespaces v1.9.1

### DISCUSSION THREAD

**Discussion by marcelokuhl** [community/community#26295] [Codespaces]
<p>Após atualizar o GitHub Codespaces para a versão v.1.9.1, não consigo mais compilar usando o CS50 (code50),<br>
Como desfazer a atualização? O makefile parece que foi alterado.<br>
OBS.: os programas funcionavam na versão anterior.<br>
<img src="https://user-images.githubusercontent.com/68967909/181096714-aca39eda-077c-487a-b8a2-4185a4295469.png" alt="image" data-base62-sha1="6i1Jt9oKKlABATzrxLHxzZHSCHn" width="163" height="46"></p>
<p><img src="https://user-images.githubusercontent.com/68967909/181096716-de4aa5f5-e6e4-4b3b-9731-f200d8c3abad.png" alt="image" data-base62-sha1="dKAIeIks0QZzLbKrKCNCqTyVNZ6" width="631" height="80"></p>

**Comment by Akash1134** [comment, marked as answer]
<p>Welcome to the GitHub community, <a class="mention" href="/marcelokuhl">@marcelokuhl</a>  We’re happy you’re here. You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails–giving a few more details might help someone give you a nudge in the right direction. I’ve gone ahead and moved it for you. Good luck!</p>
